src/ZendeskApi.Client/Logging/ILogAdapter.cs

```diff
@@ -1,15 +0,0 @@
-namespace ZendeskApi.Client.Logging
```

Comment:
I need to include logging scopes on the api calls. totally on the map. I would also like to include the diagnostics api too.